### PR TITLE
feat: add skeleton loader for modal connecting state

### LINKED ISSUE #34: Add skeleton loader for modal connecting state
Currently shows just a spinner which doesn't look good and causes the modal height to jump. Instead it should show the connected content, with skeleton loaders for any content that is loading.

### PR DESCRIPTION
**Pull request by hirenchauhan2**
Closes #34

### CODE REVIEW

**Comment by hirenchauhan2** [comment]
Hi @rolznz,
 
I just noticed, you haven't added `hacktoberfest` topic on this project. Could you please add that topic to mark this PR accepted for Hacktoberfest?

File: src/components/bc-start.ts [hunk 1]
```diff
@@ -60,7 +76,7 @@ export class Start extends withTwind()(BitcoinConnectElement) {
               How would you like to connect?
             </h1>
 
-            <bc-connector-list />
+            <bc-connector-list></bc-connector-list>
```
Comment (rolznz) [inline src/components/bc-start.ts:79]: thanks for fixing this too :pray:

**Review by rolznz — APPROVED** [review]
tACK, great job!

**Comment by rolznz** [comment]
@hirenchauhan2 thanks very much. I added the hacktoberfest and hacktoberfest2023 tags to the project

**Comment by rolznz** [comment]
@hirenchauhan2 do you have a bitcoin lightning address and nostr account? 

We will consider you for a top Alby contributor for this year's hacktoberfest

CC @itstomekk

**Comment by hirenchauhan2** [comment]
@rolznz , unfortunately no. I'm not into Bitcoin/crypto. Have lost a lot into that 😭

**Comment by MoritzKa** [comment]
No worries. There are indeed some bad crypto projects out there. It is even more remarkable that you made this PR. 🙌

File: src/components/bc-start.ts (hunk 1 above)
Comment (hirenchauhan2) [reply src/components/bc-start.ts:79]: I saw this when I ran `npm run lint`. The other error was in the test-file for `my-component`. I was not sure what do there hence did not fixed that.

**Comment by hirenchauhan2** [comment]
Let me know if you need any help on the front-end side. I'll do my best!

**Pull request merged**